### ISSUE TITLE
howto reference action of custom integration?

### ISSUE THREAD

**Issue by coderofsalvation** [DataFire/DataFire#25]
I love how the YAML can reference actions from integrations, this promotes re-use of actions, instead of hardcoding methods into the integration.

    paths:
        "/db/:id":
            get:
                action: mongodb/findOne

But, how can i reference an action of my own custom integration?

    paths:
        "/db/:id":
            get:
                action: mongodb/findOne
            post:
                action: ./myfile.js/mypostAction

This does not work :/ Probably because it applies `require()` on it.
How about an optional spaceseparated action? something like this:

    paths:
        "/db/:id":
            get:
                action: mongodb/findOne
            post:
                action: ./myfile.js mypostAction

or this:

    paths:
        "/db/:id":
            get:
                action: mongodb/findOne
            post:
                file: ./myfile.js
                action: mypostAction

**Comment by bobby-brennan** [comment]
If you create your custom integration in the `./integrations` directory, you can just reference it by it's ID.  For example, if your integration is in `./integrations/petstore/index.js`, you can reference its actions as `petstore/getPetById`.

That said, it'd be better not to force users to use a particular directory structure to organize their integrations.  Maybe the DataFire.yml config should have an `integrations` section which pulls in custom integrations, e.g.

```yaml
integrations:
  petstore: ./petstore-integration.js

paths:
  /rover:
    get:
      action: petstore/getPetById
      input:
        petId: 23
```

**Comment by bobby-brennan** [comment]
This feature is now available in 2.6.0.
